src/engine/Operation.cpp

```diff
@@ -246,9 +246,12 @@ void Operation::updateRuntimeInformationWhenOptimizedOut(
   // To set it to zero we thus have to set the `totalTime_` to this sum.
```

Comment:
The comment says "To set it to zero", but it's unclear why we would want that. Also, this comment looks like it needs reformatting

Reply:
I have added a clarifying introdution, and the formatting looks odd but is fine, because we want the "mathematical"
expression in one line.